source/includes/_users.md

```diff
@@ -175,10 +209,14 @@ This endpoint create an user and return it
 
 Parameter | Type | Description
 --------- | ---- | -----------
-name | string | The User's name.
+first_name | string | The User's first name.
+last_name | string | The User's last name.
 email | string | The User's corporate email.
 phone | string | The User's phone.
 password | string | The User's password.
+is_admin | bolean | The User's is_admin.
```

Comment:
specifies whether the user you are creating is an administrator or not

Reply:
done